types/types.pb.go

```diff
@@ -40,10 +39,10 @@ It has these top-level messages:
 	Validator
 	KVPair
 */
-//nolint: gas
```

Comment:
oh ya, we also need to preserve this, since the generated code violates some rules of the linter.

maybe we can expand our tool to include the `nolint` statements we want?